### DISCUSSION TITLE
I am not able to install backpack/pro even i puchase backpack/pro latest version

### DISCUSSION THREAD

**Discussion by shilpitrusty** [Laravel-Backpack/community-forum#915] [Q&A (Help)] [answered]
I am not able to install backpack/pro even I purchased backpack/pro latest version.

**Comment by pxpm** [comment, marked as answer]
Hey @shilpitrusty 

I think you are missing some steps in the `Intructions` tab (the last tab in the page you showed). 

You should run a command to create your authentication credentials, then add our private repository to your composer and require the package. 

It seems you missed at least the add repository to composer step, as it's trying to get the package from github and not from the private repository. 

Let me know if that helps.

Cheers

**Reply by shilpitrusty** [reply]
Issue fixed. Thank you
